lib/rest-client.js

```diff
@@ -22,17 +22,52 @@ module.exports = class RestClient {
     return await this.request.post('/open-api/order/create', params);
   }
 
+  async placeActiveOrderV2(params) {
```

Comment:
I'd suggest to replace `placeActiveOrder` method with this one. The [the docs](https://github.com/bybit-exchange/bybit-official-api-docs/blob/master/en/rest_api.md#active-order) recommend to use v2 method over v1.

I would then publish a new minor or even major version of the module.